bittensor/axon.py

```diff
@@ -888,25 +890,38 @@ async def default_verify(self, synapse: bittensor.Synapse):
             # Build the unique endpoint key.
             endpoint_key = f"{synapse.dendrite.hotkey}:{synapse.dendrite.uuid}"
 
-            # Check the nonce from the endpoint key with 4 second delta
-            allowedDelta = 4000000000
-
             # Requests must have nonces to be safe from replays
             if synapse.dendrite.nonce is None:
                 raise Exception("Missing Nonce")
 
             # If we don't have a nonce stored, ensure that the nonce falls within
             # a reasonable delta.
+
+            # Updated nonce using NTP implementated at v7.2
             if (
-                self.nonces.get(endpoint_key) is None
-                and synapse.dendrite.nonce <= time.time_ns() - allowedDelta
-            ):
-                raise Exception("Nonce is too old")
-            if (
-                self.nonces.get(endpoint_key) is not None
-                and synapse.dendrite.nonce <= self.nonces[endpoint_key]
+                synapse.dendrite.version is not None
+                and synapse.dendrite.version >= V_7_2_0
             ):
-                raise Exception("Nonce is too old")
+                # If we don't have a nonce stored, ensure that the nonce falls within
+                # a reasonable delta.
+                current_time = BittensorNTPClient.get_current_ntp_time()
+                if (
+                    self.nonces.get(endpoint_key) is None
+                    and synapse.dendrite.nonce <= current_time - ALLOWED_DELTA
```

Comment:
```suggestion
                    and synapse.dendrite.nonce <= current_time - ALLOWED_DELTA - (synapse.timeout or 0)
```

When transferring large requests there is a lot of delay (both on the wire and serialization/deserialization).
Without taking `timeout` parameter into account here this change will affect the slower axons in at least one of the subnets I know.

Reply:
Thanks. We've added timeout in our calculation but kept delta the same for now. 
We'll keep monitoring this in testnet and see if it needs any adjustments